src/IB/IBFEMethod.cpp

```diff
@@ -840,6 +882,7 @@ IBFEMethod::midpointStep(const double current_time, const double new_time)
     const double dt = new_time - current_time;
     for (unsigned int part = 0; part < d_num_parts; ++part)
     {
+        if (!d_part_is_active[part]) continue;
```

Comment:
Let's not include this guard --- I think that I would like for all computations to proceed "as usual" for inactivated parts, except that at the end of the day, forces are not spread and velocity is not interpolated.